src/Mod/Sketcher/App/SketchObject.h

```diff
@@ -63,6 +63,11 @@ class SketcherExport SketchObject : public Part::Part2DObject
     SketchObject();
     ~SketchObject();
 
+    // Many of the methods here use geoId and posId to point to sketch elements.  geoIds are
+    // indexes into the list returned by getInternalGeometry(), while posIds are PointPos values
+    // that indicate whether you're talking about a line, the start or endpoint of a line, a bare
+    // vertex, etc.  (See PointPos definition for more on how PointPos is used).
+

```

Comment:
I tend not to use "internal geometry", because this is the name I use for the "Internally aligned geometry" (like the circles at pole position of a BSpline or the major or minor axis of an ellipse). However, It is ok to use internal, as from the context, the other meaning does not apply). No change required for this.